src/chromeDebugAdapter.ts

```diff
@@ -145,9 +146,17 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
     }
 
     protected onFrameNavigated(params: Crdp.Page.FrameNavigatedEvent): void {
-        if (params.frame.url === this._userRequestedUrl) {
+        const url = params.frame.url;
+        const requestedUrlNoAnchor = this._userRequestedUrl.split('#')[0]; // Frame navigated url doesn't include the anchor
+        if (url === requestedUrlNoAnchor || decodeURI(url) === requestedUrlNoAnchor) { // 'http://localhost:1234/test%20page' will use the not decoded version, 'http://localhost:1234/test page' will use the decoded version
             // Chrome started to navigate to the user's requested url
-            this.events.emit(ChromeDebugSession.FinishedStartingUpEventName);
+            this.events.emit(ChromeDebugSession.FinishedStartingUpEventName, { requestedContentWasDetected: true } as FinishedStartingUpEventArguments);
```

Comment:
Do these all need a cast?

Reply:
I use the cast to make sure I'm sending the right properties.
If I don't put the cast, the object is of type any, and anything is considered valid.
If I put the cast, then the compiler will fail if I don't put the valid properties here.

Reply:
I was thinking 'emit' was typed here